blackjack.py

```diff
@@ -201,12 +227,19 @@ def __str__(self):
             print(response)
         #NEED TO IMPLEMENT SPLIT
         else:
-            response = my_input("Type h for hit, s for stand, or g for split.\n")
-    again_input = "Game is over.\nType yes to play again. "
+            response = my_input("Type h for hit, s for stand.\n")
+    time.sleep(0.3)
+    if(total_cash <= 0):
+        print("\nGame over. You are out of money.")
+        break
+    time.sleep(0.3)
+    again_input = "\nRound over. You have $"+str(total_cash)+".\nType yes to play again. "
```

Comment:
IMO, you should make this "type END" to end the game. Hit Enter to continue. Too easy to end the game otherwise.